ctest.h

```diff
@@ -391,7 +419,7 @@ static void color_print(const char* color, const char* text) {
         printf("%s\n", text);
 }
 
-#ifdef __APPLE__
+#if defined (__APPLE__) && !defined (_POSIX_C_SOURCE)
```

Comment:
I'm not sure if it's the same on Linux. On that note, why is setup/teardown feature for Apple only? I don't see Apple-only functions in `find_symbol`